types/wallop/index.d.ts

```diff
@@ -0,0 +1,112 @@
+// Type definitions for Wallop 2.4
+// Project: http://pedroduarte.me/wallop
```

Comment:
It should be a project url here

Reply:
My apologies, that's the one `dts-gen` added automatically. Changed.